harness/determined/pytorch/samplers.py

```diff
@@ -105,9 +108,9 @@ def __init__(
         self.rank = rank
 
     def __len__(self) -> int:
-        full_global_batches = len(self.batch_sampler) // self.num_workers
-        worker_gets_partial_batch = int(len(self.batch_sampler) % self.num_workers > self.rank)
-        return full_global_batches + worker_gets_partial_batch
+        full_global_batches, extra_batches = divmod(len(self.batch_sampler), self.num_workers)
```

Comment:
wow, every time I think I know all the builtins in python, somebody shows me a new one.